From b67a47e67dc7d40ba165859b1885eb9aa961bea6 Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Wed, 22 Feb 2012 14:11:56 +0100
Subject: [PATCH 040/109] scsi-disk: enable CD emulation

RH-Author: Paolo Bonzini <pbonzini@redhat.com>
Message-id: <1329919979-20948-40-git-send-email-pbonzini@redhat.com>
Patchwork-id: 37518
O-Subject: [RHEL 6.3 qemu-kvm PATCH v2 039/102] scsi-disk: enable CD emulation
Bugzilla: 782029
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>

RHEL only.
---
 hw/scsi-disk.c |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/scsi-disk.c |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/hw/scsi-disk.c b/hw/scsi-disk.c
index e80a366..8de99e2 100644
--- a/hw/scsi-disk.c
+++ b/hw/scsi-disk.c
@@ -1258,11 +1258,6 @@ static int scsi_initfn(SCSIDevice *dev, uint8_t scsi_type)
     SCSIDiskState *s = DO_UPCAST(SCSIDiskState, qdev, dev);
     DriveInfo *dinfo;
 
-    if (scsi_type == TYPE_ROM) {
-        error_report("scsi-disk: cdrom emulation is not supported");
-        return -1;
-    }
-
     if (!s->qdev.conf.bs) {
         error_report("drive property not set");
         return -1;
-- 
1.7.7.6

